### PR TITLE
libomp: update to 8.0.1; bump libomp-devel

### PR DESCRIPTION
**Pull request by eborisch**
Maintainer update.

### CODE REVIEW

File: lang/libomp/Portfile [hunk 1]
```diff
@@ -31,40 +30,42 @@ if { ${subport} eq "libomp-devel" } {
...
-        version                 349521
+        # Moved to official llvm github project. It would be a very large 
+        # (100MB+) download, so use the svn interface (!) to github to 
+        # download just the openmp directory. Not using LLVM's own SVN since
+        # they are in the process of migrating to github.
+        fetch.type              svn
+        svn.url                 https://github.com/llvm/llvm-project/trunk/openmp
+        svn.revision            327229
+        # llvm-svn revision; also used for livecheck
+        version                 367070
```
Comment (mf2k) [inline lang/libomp/Portfile]: This is a non-sensical `version`. At least use a date format (e.g. `2019-07-26`) so it can increase going forward.

Comment (mf2k) [reply lang/libomp/Portfile]: Is this the build count? AFAIK, no other port uses a version numbering scheme like that.

Comment (eborisch) [reply lang/libomp/Portfile]: How do you figure? This is the upstream SVN revision number. Now, I’ll give you that the two SVNs here makes it confusing (the upstream SVN and the SVN # from github’s SVN interface), but I’m continuing to use the SVN number from the still-primary upstream SVN as the version number for this (head-tracking) -devel version. (At least until they fully move to GH.

Comment (mf2k) [reply lang/libomp/Portfile]: Well I suppose if it increases it is fine.

**Comment by pmetzger** [comment]
The CI failed; I'm reluctant to merge without knowing why.

**Comment by eborisch** [comment]
I agree, but it seems to be only the -devel versions failing; I don't know why these are, as I don't seem to know enough about the CI to find logs saying why they failed. I can install it in sandbox mode just fine on my 10.14 box; is anyone else able to test the patch to see in they can recreate a failure (or success)?

**Comment by mf2k** [comment]
libomp-devel is failing in the post-extract phase.

```
--->  Patching check-depends.pl: /bulk.*compatibility/s/s+/s+.*/
DEBUG: Executing reinplace: /usr/bin/sed /bulk.*compatibility/s/s+/s+.*/ </opt/local/var/macports/build/_Users_travis_build_macports_macports-ports_lang_libomp/libomp-devel/work/libomp-devel-367070/runtime/tools/check-depends.pl >@file11
DEBUG: couldn't read file "/opt/local/var/macports/build/_Users_travis_build_macports_macports-ports_lang_libomp/libomp-devel/work/libomp-devel-367070/runtime/tools/check-depends.pl": no such file or directory

```

**Comment by pmetzger** [comment]
Okay, so it's a real bug...

**Comment by eborisch** [comment]
Oh... it's from me running the pre-release macports. I'll fix it up.

**Comment by eborisch** [comment]
For reference, I was relying on the new auto-magic symlink on extract code. Adding actual worksrcdir path fixed the issue.

**Comment by pmetzger** [comment]
@eborisch I'm guessing since @mf2k is okay with this, it builds, and you're the maintainer, that you can commit it?

**Pull request merged**